vite.config.mts

```diff
@@ -18,7 +18,7 @@ export default defineConfig({
             interopDefault: true
         },
         include: ['**/test/**/*.test.ts'],
-        exclude: ['**/node_modules/**', '**/dist/**', '**/generated/**', '**/templates/**'],
+        exclude: ['**/node_modules/**', '**/dist/**', '**/generated/**', '**/templates/**', '**/examples/hello*/**'],
```

Comment:
That's most likely not related to this PR, right?
Is the code `examples/hello*/` a left-over of a previous run of `generator-langium` tests?

Reply:
Not necessarily, but I've found that the `npm run test` command restarts once the yeoman generator test has passed, since it updates the files. Adding this exclude rule seems to fix it.